src/content/docs/codestream/start-here/sign-up-codestream.mdx

```diff
@@ -3,48 +3,61 @@ title: Sign up for CodeStream
+IDE.
 
 ## Create an account [#account]
 
-If you already have the CodeStream extension [installed in your IDE](/docs/codestream/start-here/install-codestream), you can start the sign up process from the CodeStream pane.
+If you already have the CodeStream extension [installed in your
+IDE](/docs/codestream/start-here/install-codestream), you can start the sign up
+process from the CodeStream pane.
 
 ![Create an account](./images/CreateAnAccount6.png)
 
-There are two ways to sign up for CodeStream. You can sign up with a set of CodeStream credentials (such as, email address and password). Alternatively, you can sign up using your GitHub, GitLab, or Bitbucket (cloud versions only) account. 
+You can sign up using your email address and a password, or you can sign up using
+your GitHub, GitLab, or Bitbucket account. If you're a New Relic customer, you can also sign up with a user api
+key from your New Relic One account.
 
-Signing up via your code host also connects your repositories to CodeStream. If the email address you're using with your code host isn't your work email, you should create CodeStream-specific credentials instead.
+If the email address you're using with your code host isn't your work email, you
+should sign up with your work email address instead.
 
-If you sign up via CodeStream, your next step will be to confirm your email address by entering a code sent to you via email. You can paste the code into any of the boxes rather than typing each number individually.
+Once you've signed up, we'll send a confirmation code to your email.
 
 ![Confirm Email](./images/EmailConfirmation1.png)
 
-## CodeStream organizations [#organizations]
-
-A CodeStream organization is where you and your teammates will discuss code. Similar to a Slack workspace, all of the developers in your company are in the same CodeStream organization. CodeStream's activity feed keeps things relevant for you by showing activity related to the code you have open in your IDE. 
-
-The discussions about code build up a knowledge base that is a company-wide resource, so the only reason to have multiple organizations on CodeStream is if you truly need separation. For example, you might have an organization for your day job and another for an open-source project you work on. Or maybe you're a consultant that's a member of different CodeStream organizations for each of your clients.
-
 ## Create or join an organization [#create-org]
 
-If you're invited to join an organization on CodeStream, sign up with the same email address the invitation was sent to. You'll automatically be added to that organization.
+A CodeStream organization is where you and your teammates will discuss and
+collaborate on code, and all of the developers in your company can be in the
+same CodeStream organization. If you're a New Relic customer, note that your
+CodeStream organization is not connected in any way to your organization on New
+Relic One.
```

Comment:
I might word this as "not connected in any way to your New Relic organization". Instead of referencing 'New Relic One'. And maybe link to https://docs.newrelic.com/docs/accounts/accounts-billing/account-structure/new-relic-account-structure/ if you think it helps.